FirebaseCore/Tests/Unit/FIRAnalyticsConfigurationTest.m

```diff
@@ -16,7 +16,7 @@
 
 #import "FirebaseCore/Tests/Unit/FIRTestCase.h"
 
-#import <FirebaseCore/FIRAnalyticsConfiguration.h>
+#import "FirebaseCore/Sources/FIRAnalyticsConfiguration.h"
```

Comment:
May be a silly question but just to confirm for here and other tests - is it alright that we're not using framework imports in test code?

Reply:
Yes. The tests are in the repo and using repo-relative imports should allow Swift Package Manager to build them without change. We should leave framework import testing for integration tests.